app/src/main/res/layout/home_tile.xml

```diff
@@ -59,4 +59,10 @@
             android:textColor="@android:color/black"
             tools:text="YouTube" />
     </LinearLayout>
+
+    <View
```

Comment:
nit: Add a comment explaining why this is necessary.

Reply:
https://github.com/boek/firefox-tv/blob/85acba3a056743da561046030fcae775597facfa/app/src/main/res/layout/home_tile.xml#L63